src/pages/Home/homepage.css

```diff
@@ -56,4 +53,102 @@
         height: 100%;
         width: 100%;
     }
+}
+ /* ------------------------------------------------- */
+ .banner-boxy{
+     /* border: 1px solid black; */
+     height: 50rem;
+ }
+.banner-text{
+    /* border: 1px solid black; */
+    display: flex;
+    flex-direction: column;
+    gap: 4rem;
+    align-items: center;
+    justify-content: center;
+}
+.banner-heading{
+    margin-top: 5rem;
+    font-family: 'Lato', sans-serif;
```

Comment:
You've used this font-family many times in your code, you can make a utility class for this for just use variable.